### DISCUSSION TITLE
SMB Data dir

### DISCUSSION THREAD

**Discussion by Marc-Proux** [nextcloud/all-in-one#4565] [Questions] [answered]
I'm new to using Nextcloud AIO and I have a question about this section:
https://github.com/nextcloud/all-in-one?tab=readme-ov-file#can-i-use-a-cifssmb-share-as-nextclouds-datadir

Can I set up the SMB share on my user folder so that I can access the files directly via SMB on my Windows PC and also create/edit files? I'm asking this because many forums suggest that using an SMB share with Nextcloud files is not ideal, as changes are logged in a database and direct use of an SMB file could disrupt Nextcloud's functionality. Additionally, I tried editing a file directly on my server console, but the changes were not reflected in Nextcloud.

Thanks for the help.

**Comment by szaimen** [comment, marked as answer]
Hi, please ask your question here: https://help.nextcloud.com
